src/app/pages/your-virtrolio/your-virtrolio.component.html

```diff
@@ -138,3 +153,37 @@ <h5 class="modal-title" id="send-message-modal-title">Signing a friend's virtrol
     </div>
   </div>
 </div>
+
+<!-- Share your link modal -->
+<div class="modal fade" id="share-link-modal" tabindex="-1" role="dialog" aria-labelledby="share-link-modal"
+     aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="share-link-label">Send your Sharing Link</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <div class="row justify-content-center px-4" id="sharing-buttons">
+          <button role="button" class="btn btn-lg col-sm-3 mx-sm-2 my-2 my-sm-0 btn-secondary" id="facebook-button"
+             (click)="postOnSocial('facebook')">
+            <img src="../../../assets/images/icons/facebook.svg"> Facebook
+          </button>
+          <button role="button" class="btn btn-lg col-sm-3 mx-sm-2 my-2 my-sm-0 btn-secondary" id="twitter-button"
+             (click)="postOnSocial('twitter')">
+            <img src="../../../assets/images/icons/twitter.svg"> Twitter
```

Comment:
Missing required alt attribute
```suggestion
            <img src="../../../assets/images/icons/twitter.svg" alt="Share with Twitter"> Twitter
```